standards/sql-standards.md

```diff
@@ -8,38 +8,308 @@ formatting issues and arguments in merge requests. The resulted in a Shed
 specific set of rules that plug and play nicely with the tool
 [SQLFLUFF](https://www.sqlfluff.com/).
 
-Add the configuration file called `sqlfluff.config` to your project directory or
+Add the configuration file called `.sqlfluff` to your project directory or
 somewhere referenced/linked to in your code editor of choice. The configuration
 setting:
 
 ```toml
```

Comment:
This section's getting pretty hefty now: worth refactoring out into an actual TOML file?

Reply:
Maybe? You mean have it separate with a link to it on this page instead?

Reply:
Yeah, link to an external file but any particular parts that need to be documented can be called out in here…?

Reply:
Done that now, should be in the latest commit